default.nix

```diff
@@ -1,87 +1,99 @@
+########################################################################
+# default.nix -- The top-level nix build file for plutus.
+#
+# This file defines an attribute set of cardano-sl packages.
+#
+# It contains:
+#
+#   - pkgs -- the nixpkgs set that the build is based on.
+#   - haskellPackages.* -- the package set based on stackage
+#   - haskellPackages.ghc -- the compiler
+#   - plutusPackages -- just plutus packages
```

Comment:
I propose we just call it `localPackages` or something

Reply:
`projectPackages`?